Models/DTO/Administrator/GetDataTDTO.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Newtonsoft.Json;
+
+namespace API.Models.DTO.Administrator
+{
+    public class GetDataTDTO<T>
+    {
+        [JsonProperty("meta")] public Meta Meta { get; set; }
+        [JsonProperty("data")] public T Data { get; set; }
+
+        public GetDataTDTO(T orders)
+        {
+            Meta = new Meta();
+            Data = orders;
```

Comment:
Will `T` always be orders? Naming is inconsistent here.

Reply:
Any kind of object.

Reply:
My mistake.